### DISCUSSION TITLE
DNS clients > new criteria > IP addr.range.

### DISCUSSION THREAD

**Discussion by Yuri-01** [AdguardTeam/AdGuardHome#3093] [Q&A]
Add ablity to specify DNS client by ip addr.range.

For example:
192.168.0.100-192.168.0.199

Currently I should be added individual addresses several times
For example:
step #1: add 192.168.0.100
step #2: add 192.168.0.101
...
step #100: add 192.168.0.199

**Comment by ameshkov** [comment]
You can use CIDRs instead: `192.168.0.0/24`

**Reply by Yuri-01** [reply]
Hi, Andrew.

I know, that I can specify group of DNS clients by IP **network** (in CIDR notation).

By my suggestion - is specify group of DNS clients by IP **range** at the once.

For example above:
I have LAN 192.168.0.0/24
And I want to subdivide some clients with IP addr.from range 192.168.0.100-192.168.0.199 as group of DNS clients "cli_somename-01" and some other clients with IP addr.from range 192.168.0.200-192.168.0.250 as group of DNS clients "cli_somename-02" 
How can I specify it's ranges in CIDR notation?

**Reply by ameshkov** [reply]
You can't do it THAT precisely, but you can do something like this:

* `192.168.0.0/25` - `192.168.0.0-192.168.0.127`
* `192.168.0.128/25` - `192.168.0.127-192.168.0.255`

Try using this to see what ranges you can use: https://www.ipaddressguide.com/cidr

**Reply by Yuri-01** [reply]
Thank you, Andrew.

Therefore I and suggest this feature.
(Because specify IP range through IP network (in CIDR notation) is not are fully equvalen).
